cmd/cloud_sql_proxy/cloud_sql_proxy.go

```diff
@@ -536,6 +554,17 @@ func main() {
 		RefreshCfgThrottle: refreshCfgThrottle,
 	}
 
+	if *performConnectivityTestsOnStartup == true {
+		for _, instanceConfig := range cfgs {
+			err := checkInstanceConnectivity(instanceConfig.Instance, instanceConfig.IsPublic, proxyClient);
+			if err != nil {
+				log.Fatal(err)
```

Comment:
Use the `logging` package?

followed by
`os.Exit(...)`

Reply:
We use `fatal` in the code all over. I count almost no errors and `os.Exit`, especially not used that way.

Could you let me know if you think that's the best way to proceed still?